ui/less/overflow_menu.less

```diff
@@ -67,13 +67,6 @@
     padding-left: 10px;
     padding-right: 10px;
   }
-
-  /* If the seekbar is missing, this is positioned lower.
-   * TODO: Solve with flex layout instead? */
-  &.shaka-low-position {
```

Comment:
Any specific reason we have to remove this style? It might break the current behavior with no seekbar.
The most ideal solution might be to use the flex layout for the overflow menu to let it float above the "shaka-controls-button-panel", however it requires further investigation to make sure nothing would be broken.

Reply:
this style was actually present because the bottom property was set to static 30px which had to be changed when there was no seekbar , but now as I changed the bottom property to the 4em (relative measurement) it is not needed as it automatically gives required margin from bottom making sure it is above the controls even without the seekbar, I verified that the change doesn't break the current behavior. below are the images of UI  without seek bar before and after these changes are made.

before:

![before](https://user-images.githubusercontent.com/57630117/111181230-c3881080-85d3-11eb-8fad-d8d64243fccb.png)

after: 

![after](https://user-images.githubusercontent.com/57630117/111181411-f4684580-85d3-11eb-8128-ad7fba90bf30.png)


And you are right, the optimal and permanent solution would be to use a flex layout for the overflow menu to let it float above the control panel, but these changes should work as a good temporary solution.

Reply:
Thank you for verifying that!
Would you like to try out the flex layout approach, since it's not an urgent fix? We can come back to the current approach if it doesn't work.

Reply:
Sure sir, I'll try to implement the flex layout and see if it works.